src/app/containers/Index/Title/index.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import Title from '.';
+
+describe(`Title`, () => {
+  describe('snapshots', () => {
+    shouldMatchSnapshot(
+      'should render visual title',
+      <ServiceContextProvider service="igbo">
+        <Title>Title</Title>
+      </ServiceContextProvider>,
+    );
+
+    shouldMatchSnapshot(
+      'should render visually hidden title',
+      <ServiceContextProvider service="igbo">
+        <Title isVisuallyHidden>Title</Title>
+      </ServiceContextProvider>,
+    );
+
+    shouldMatchSnapshot(
+      'should render title without service provided',
+      <Title>Title</Title>,
```

Comment:
I think it would be nice to have assertions here instead of snapshot. It would make it easy to know what is being tested than having to go to the snapshot files. Which also makes it resilient to style changes.

Reply:
Yeah I think that's a good point - I've been used to using shallow rendering + snapshots for this kind of test previously, but this is overkill with a full mount